### PR TITLE
Fixed version of the navigation fixes added in the prior PR.  The goa…

### PR DESCRIPTION
**Pull request by jberkus**
…l of these changes is to move the content that helps people get up and running with podman, and with the podman community, to be the most obvious content.

* Moved Mailinglist to "Community" and added information about Slack and IRC
* Created "Getting Started" and moved installation instructions and initial usage tutorial there, copying them from libpod
* Corrected "blogs" to "blog"
* Moved Getting Started and Community to the top of the menu

Replaces PR #126, which had inadvertent testing artifacts in it.

attn @TomSweeneyRedHat

### CODE REVIEW

File: community/index.html [hunk 1]
```diff
@@ -1,12 +1,22 @@
...
+We want your feedback, issues, patches, and involvement in the development of Podman.  See Contributing for how to submit issues & PRs, or better chat with us on Slack, IRC, or Email.
+
+<h2>Slack and IRC</h2>
+
+The Podman developers often hang out on the #CRI-O channel in [Kubernetes Slack](https://slack.k8s.io/).  Note that the devs are generally around during CEST and Eastern Time business hours, so be patient if you're in another time zone.
+
+There's also Podman users and developers on [IRC.Freenode.Net](https://webchat.freenode.net/) channel #podman.
+
+<h2>Mailing List</h2>
+
```
Comment (TomSweeneyRedHat) [inline community/index.html:19]: Really nice change @jberkus

File: community/index.html [hunk 2]
```diff
@@ -1,12 +1,22 @@
...
 layout: default
-title: Podman Mailing List  
+title: Join Our Community
 ---
 
 <img src="https://podman.io/images/podman.svg" alt="Podman logo">
 
 <h1>{{ page.title }}</h1>
 
+We want your feedback, issues, patches, and involvement in the development of Podman.  See Contributing for how to submit issues & PRs, or better chat with us on Slack, IRC, or Email.
```
Comment (TomSweeneyRedHat) [inline community/index.html]: s/better yet chat/better chat/ perhaps?  Or something other then "better chat"

File: getting-started/installation.md [hunk 3]
```diff
@@ -0,0 +1,445 @@
...
+A proper description of setting up CNI networking is given in the [`cni` README](cni/README.md).
+
+Using the CNI plugins from above, a more basic network config is achieved with:
+
+```bash
+sudo mkdir -p /etc/cni/net.d
+curl -qsSL https://raw.githubusercontent.com/containers/libpod/master/cni/87-podman-bridge.conflist | sudo tee /etc/cni/net.d/99-loopback.conf
+```
+
+
```
Comment (TomSweeneyRedHat) [inline getting-started/installation.md:272]: Extra line here.  Also I've a PR up (https://github.com/containers/libpod/pull/3877) that's changing this section in particular a bit, plus a few other spots.

File: getting-started/getting-started.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+```
+You can verify the deletion of the container by running *podman ps -a*.
+
+## Integration Tests
+For more information on how to setup and run the integration tests in your environment, checkout the Integration Tests [README.md](../../test/README.md)
+
+## More information
+
+For more information on Podman and its subcommands, checkout the asciiart demos on the [README.md](../../README.md#commands)
+page.
```
Comment (TomSweeneyRedHat) [inline getting-started/getting-started.md:150]: Any changes for this or the install guide, or just a pullover and name change from libpod?  I think just a pull over.

**Comment by TomSweeneyRedHat** [comment]
@jberkus, I like the approach a LOT!  I found one minor grammatical thing and had a couple of informational questions.   @rhatdan WDYT?

File: community/index.html (hunk 2 above)
Comment (jberkus) [reply community/index.html]: replaced with a simple "and chat with us"

File: getting-started/getting-started.md (hunk 4 above)
Comment (jberkus) [reply getting-started/getting-started.md:150]: yeah, it's just a pull over.  There's apparently some other content that needs to be re-linked, based on the move; more likely I need to copy over.

File: getting-started/installation.md (hunk 3 above)
Comment (jberkus) [reply getting-started/installation.md:272]: currently the only difference between the files is the Jekyll header, so we can just copy.

**Comment by TomSweeneyRedHat** [comment]
LGTM, going to merge.  Thanks @jberkus

**Pull request merged**